TRRandomizerCore/TRRandomizerController.cs

```diff
@@ -1167,6 +1167,41 @@ public bool EnableTr2Jumping
         }
     }
 
+    public bool FixBearAi
+    {
+        get => ScriptEditor is TR1ScriptEditor tr1Editor && tr1Editor.FixBearAi;
+        set
+        {
+            if (ScriptEditor is TR1ScriptEditor tr1Editor)
```

Comment:
You can do `(ScriptEditor as TR1ScriptEditor)?.FixBearAi = value;` if you liked, same with the others.

Reply:
No you can't, not for assignment. I was sure you could, maybe I just really wanted it.